modules/nw-configuring-high-performance-multicast-with-sriov.adoc

```diff
@@ -1,19 +1,18 @@
 // Module included in the following assemblies:
 //
-// * networking/multiple_networks/configuring-sr-iov.adoc
+// * networking/hardware_networks/using-sriov-multicast.adoc
 
 [id="nw-configuring-high-performance-multicast-with-sriov_{context}"]
 = Configuring high performance multicast
 
-Openshift SDN supports multicast between Pods on the default network. This is best used for low-bandwidth coordination or service discovery, and not a high-bandwidth solution. For applications of streaming media, such as IPTV and
-multipoint videoconferencing, you can utilize SR-IOV to provide near-native 
-performance.
+The Pod network provider supports multicast between Pods on the default network. This is best used for low-bandwidth coordination or service discovery, and not high-bandwidth applications.
```

Comment:
Where is `Pod network provider` defined?

Reply:
@zshi-redhat, I was trying to future proof for the switch to OVN-Kubernetes; I can change it back to OpenShift SDN and update once OVN-Kubernetes is GA.

Reply:
@jboxman thanks for the explain! ack.